### ISSUE TITLE
ETQ instructeur, un choix oui/non apparaît pour la newsletter hebdomadaire, alors que le choix "oui" est défini par défaut et non modifiable 

### ISSUE THREAD

**Issue by benjaminhenkel** [betagouv/demarches-simplifiees.fr#3972]
![Screenshot 2019-06-18 at 17 42 06](https://user-images.githubusercontent.com/28590367/59698752-a81a8680-91f0-11e9-8a0e-1cf44deb2a49.png)

**Comment by benjaminhenkel** [comment]
# 18097

**Comment by kemenaran** [comment]
@Benjamin-Doberset oui, c'est fait exprès, pour indiquer ce qu'on envoie de toute façon. Histoire que les gens comprennent ce qu'il vont recevoir et pourquoi.

Quel est l'objet de l'issue exactement ? C'est perturbant ? La personne voudraient le désactiver ?

**Comment by kemenaran** [comment]
@Benjamin-Doberset oui, c'est fait exprès, pour indiquer ce qu'on envoie de toute façon. Histoire que les gens comprennent ce qu'il vont recevoir et pourquoi.

Quel est l'objet de l'issue exactement ? C'est perturbant ? La personne voudraient le désactiver ?

**Comment by benjaminhenkel** [comment]
La personne voudrait pouvoir le désactiver, ce qui me semble une action qu'elle devrait pouvoir faire. A mon avis, les personnes qui ne veulent pas ce mail, ne le liront pas de toute manière ! Donc autant economiser du CO2 !

**Comment by benjaminhenkel** [comment]
HS 22789
